wordpress-mobile/WordPress-FluxC-Android
# Woo Shipping Labels: purchase labels

This adds support for purchasing shipping labels.

To test:
1. Create an order that satisfies shipping label creation in your store.
2. Open the example app.
3. Click on Woo -> Shipping Labels.
4. Click on purchase a shipping label.
5. Enter the order and package details.
6. Confirm that the shipping label details are printed in the console.
